src/test/java/io/openliberty/website/it/OpenLibertyEndpointIT.java

```diff
@@ -81,6 +85,23 @@ public void testNewReleaseCausesUpdate() throws InterruptedException, IOExceptio
         assertTrue(latestVersions.contains("20.0.0.5"), "The latestVersions.js should contain 20.0.0.5 as the version. Contains: " + latestVersions);
     }
 
+    @Test
+    public void testDocsVersionNeverExists() throws IOException {
+        httpURLConnection = getInvalidDocVersionResponse("https://openliberty.io/docs/22.0.0.15/overview.html");
+        int responseCode = httpURLConnection.getResponseCode();
+        BufferedReader in = new BufferedReader(new InputStreamReader(
+        httpURLConnection.getErrorStream()));
+        String inputLine;
+        StringBuilder response = new StringBuilder();
+        while ((inputLine = in.readLine()) != null) {
+            response.append(inputLine);
+        }
+        in.close();
+        String responseBody = response.toString();
+        assertEquals(500,responseCode);
+        assertTrue(responseBody.contains("com.ibm.ws.webcontainer.exception.IncludeFileNotFoundException") && responseBody.contains("SRVE0190E") && responseBody.contains("/docs/22.0.0.15/overview.html.gz"));
```

Comment:
I think we want to check if the response is 404 rather than 500.

Reply:
This pull request would need to be in production before this testcase can start checking for FileNotFound HTTP 500 maps to 404 page